utils/fixtures/balancerV2Fixture.ts

```diff
@@ -0,0 +1,92 @@
+import { Signer } from "@ethersproject/abstract-signer";
+import { JsonRpcProvider, Web3Provider } from "@ethersproject/providers";
```

Comment:
[nit] if we import from the umbrella package `ethers` rather than `@ethersproject/foo`, we can avoid issues with mismatching dependencies for ethers sub-packages. Additional context in: https://github.com/SetProtocol/set-protocol-v2/pull/144 

Concretely, this means replacing this import with
```suggestion
import { providers } from "ethers";
```

and below

```typescript
  constructor(provider: providers.Web3Provider | providers.JsonRpcProvider, ownerAddress: Address) {
```